backends/bmv2/control.cpp

```diff
@@ -400,7 +400,7 @@ ControlConverter::convertTable(const CFG::TableNode* node,
         size = BMV2::TableAttributes::defaultTableSize;
 
     result->emplace("max_size", size);
-    auto ctrs = table->properties->getProperty(BMV2::TableImplementation::directCounterName);
+    auto ctrs = table->properties->getProperty(BMV2::TableAttributes::countersName);
```

Comment:
Are you suggesting replacing them with string constant literals?  Or something else?